Dockerfile

```diff
@@ -376,7 +375,4 @@ RUN set -eux; \
     # Remove tmp files from assets:precompile
     rm -rf /tmp/* tmp/* log/* .cache/*;
 
-# Set the work dir and the container entry point
-ENTRYPOINT ["/usr/bin/tini", "--"]
```

Comment:
Other container runtimes like Kubernetes have no equivalent to Docker's `--init` flag

Reply:
@jgillich Oh, I miss this part. I should revert this changes.